distribution/docs/src/main/resources/content/_configuring/exporting-existing-configurations-contents.adoc

```diff
@@ -32,7 +32,8 @@ image::exporting_configuration_step3.png[Exporting Step 3]
 
 To export the current ${branding} configuration from the ${command-console}:
 
-. Type in `migration:export <directory>`. This command creates the exported configuration file `exported-${project.version}.zip` which is saved to the specified directory. If no directory is specified it will default to `<${branding}_HOME>/exported`
+. Run the command migration:export from the ${command-console}.
+. A file named `exported-${project.version}.zip` will be created in the `exported` directory underneath `<${branding}_HOME>`, but an alternative location can be specified with the `migration:export`. Copy this file to a secure place.
```

Comment:
❓  Same question. Should we provide an example of the command with an alternative location?

Reply:
I think the help on the command is sufficient for describing the parameters.